students/chylak_malgorzata/lesson_02_flow_control/adding_factorials.py

```diff
@@ -0,0 +1,7 @@
+n = int(input())
+factorial = 1
+sum = 0
```

Comment:
Try to avoid using built-in functions. Please have a look:
https://docs.python.org/3.6/library/functions.html